src/resources/shared/openshift/setup_openstack.md

```diff
@@ -0,0 +1,6 @@
+### Setup Your OpenStack Profile
```

Comment:
This should be a title by itself, but rather included under "Create and Deploy cluster-b". See for e.g how it is done here: https://submariner.io/getting-started/quickstart/openshift/vsphere-aws/#create-and-deploy-cluster-b-on-aws